.github/workflows/test-action-build-init.yml

```diff
@@ -66,6 +71,7 @@ jobs:
           skip-init: ${{ matrix.skip-init }}
           antsibull-docs-version: ${{ matrix.antsibull-docs-version }}
           lenient: ${{ matrix.lenient }}
+          provide-links-safely: ${{ matrix.provide-links-safely }}
```

Comment:
```suggestion
          provide-link-targets: ${{ matrix.provide-links-safely }}
```

Reply:
Ok, **that** was a stupid one... Thanks for spotting this!